src/Core/MainLoop.js

```diff
@@ -92,8 +92,26 @@ MainLoop.prototype._step = function _step(view, timestamp) {
 
     // update camera
     const dim = this.gfxEngine.getWindowSize();
+    const foo = (dim.width != view.camera.width || dim.height != view.camera.height);
+
+    const animator = {
+        t: 0,
+        update: (dt, ignore) => {
+            this.t = ignore ? 0 : Math.min(1, dt / 1000 + this.t);
+
+            if (this.t < 1) {
+                if (foo) {
```

Comment:
If foo is false, you want to do nothing right? Not even increasing `this.t` I think.

Reply:
Yes that's right, do nothing if false.